src/transaction.ts

```diff
@@ -73,25 +74,41 @@ export interface KeyRange {
 }
 
 export interface ReadRequest extends RequestOptions {
-  keySet?: s.KeySet;
+  session?: string | null;
```

Comment:
I don't think the user should ever be providing the session, so we shouldn't include it.